src/Our.Umbraco.StackedContent/PropertyEditors/StackedContentPropertyEditor.cs

```diff
@@ -51,6 +84,24 @@ internal class StackPreValueEditor : PreValueEditor
 
             [PreValueField("disablePreview", "Disable Preview", "boolean", Description = "Set whether to disable the preview of the items in the stack.")]
             public string DisablePreview { get; set; }
+
+            public override IDictionary<string, object> ConvertDbToEditor(IDictionary<string, object> defaultPreVals, PreValueCollection persistedPreVals)
+            {
+                if (persistedPreVals.IsDictionaryBased)
+                {
+                    var dict = persistedPreVals.PreValuesAsDictionary;
+                    if (dict.TryGetValue("contentTypes", out PreValue contentTypes) && string.IsNullOrWhiteSpace(contentTypes.Value) == false)
```

Comment:
hey @mattbrailsford - I'm looking to move this logic over to InnerContent, but here we're specifically checking the "contentTypes" prevalue ... which is specific to StackedContent. Hmmm, what to do?

Reply:
This is where I see it getting passed off to IC. So the SC pre value locates the "collection" but then asks IC to process them.

You could create some IC "Simple" pre value editors as well that assume the same pre value setup as SC/NC. People would then just override if they need something different.